homeassistant/components/onvif/__init__.py

```diff
@@ -155,3 +158,58 @@ async def async_populate_options(hass: HomeAssistant, entry: ConfigEntry) -> Non
     }
 
     hass.config_entries.async_update_entry(entry, options=options)
+
+
+@callback
+def _migrate_camera_entities_unique_ids(
```

Comment:
```suggestion
def _async_migrate_camera_entities_unique_ids(
```

Reply:
Usually we prefix function decorated with `@callback` with `_async` to indicate they are safe to run in the event loop since we have both async and sync code in this code base